src/services/tree-sitter/__tests__/index.test.ts

```diff
@@ -1,15 +1,11 @@
+// npx jest src/services/tree-sitter/__tests__/index.test.ts
```

Comment:
This test was mocking too much, and isn't compatible with the latest version of WASM tree-sitter. I updated it appropriately.